website/content/docs/release-notes/nomad/v1-10-x.mdx

```diff
@@ -0,0 +1,147 @@
+---
+layout: docs
+page_title: Nomad v1.10.x release notes
+description: >-
+  HashiCorp Nomad version 1.10.x release notes. New features include dynamic host volumes, OIDC. Updates to the Container Storage Interface (CSI) volume delete and volume status commands. CSI volume and plugin events added to the event stream. Added UI URL hints to the end of common CLI commands and a -ui flag to auto-open them in a browser.
+---
+
+# Nomad 1.10.x release notes
+
+We are pleased to announce the following Nomad updates.
+
+## Release highlights
+
+#### Dynamic host volumes
+
+The dynamic host volumes feature brings a persistent storage option to your
+workload allocations.
+
+Nomad dynamic host volumes manage storage for stateful workloads without
+requiring a restart of the Nomad nodes to apply configuration changes. They
+provide a workload-agnostic way to specify resources and work with the `exec`,
+`java`, and `docker` task drivers. You create dynamic host volumes via the CLI
+or API and then configure the job with the
+[`volume`](/nomad/docs/job-specification/volume) and
+[`volume_mount`](/nomad/docs/job-specification/volume_mount) blocks in the job
+specification.
+
+Host volumes mount paths from the Nomad client into allocations. Nomad is aware
+of host volume availability and makes use of it for job scheduling. However,
+Nomad does not know about the volume's underlying characteristics, so you can
+use host volumes for both local persistent storage and for highly
+persistent, networked storage.
+
+Resources:
+
+- [Host volumes section](/nomad/docs/operations/stateful-workloads#host-volumes)
+  in the _Considerations for stateful workloads_ guide for an overview and
+  comparison of storage options
+- [Host volumes plugin
+  specification](/nomad/docs/concepts/plugins/storage/host-volumes) for examples
+  of how to write your own plugin to dynamically configure persistent storage on your Nomad client nodes
+- [Use Nomad dynamic host volumes to persist data for stateful workloads
+  tutorial](/nomad/tutorials/stateful-workloads/stateful-workloads-dynamic-host-volumes)
+  to learn how to create and use a dynamic host volume for persistent storage
+
+
+#### OpenID Connect (OIDC) with Proof Key for Code Exchange (PKCE)
```

Comment:
PKCE is only one of the OIDC enhancements. The larger one is Client Assertions, which I describe over in its docs PR:

https://github.com/hashicorp/nomad/pull/25375/files#diff-5881fd55b224b5b910ab8e34cf95797b1aeb9f6e0a020c5710f3c456ab3cfe4bR77-R82

I also have a WIP tutorial here, which maybe actually explains it better?
https://github.com/hashicorp/tutorials/compare/nomad-oidc-keycloak#diff-0b5e699e7ecbab10c08f61812c8e4d69e5cdfa25dea84a8ff8b0dac2762771e7R471-R479

Let me know if you think I should change either of those (they are both as-yet unmerged), or if you'd like help writing a summary here!